exodus/exodus/core/static_analysis.py

```diff
@@ -81,6 +82,15 @@ def get_app_info(self):
         return info
 
 
+def is_paid_app(handle):
```

Comment:
You should handle exception with `try` `except` because `google_play_scraper` raises an exception when the application does not exist on Google Play or when a network issue occurred.

Reply:
Done